src/pyhf/writexml.py

```diff
@@ -205,12 +205,15 @@ def build_modifier(spec, modifierspec, channelname, samplename, sampledata):
                 for a, b in np.array((modifierspec['data'], sampledata)).T
             ],
         )
+    elif modifierspec['type'] == 'shapefactor':
+        pass
     else:
         log.warning(
-            'Skipping {0}({1}) for now'.format(
+            'Skipping modifier {0}({1}) for now'.format(
```

Comment:
@kratsg It seems this isn't getting covered. https://codecov.io/gh/scikit-hep/pyhf/pull/1136/diff#D3-2111 Can we fix this up?